yaksh/templates/yaksh/course_forum.html

```diff
@@ -0,0 +1,90 @@
+{% extends "user.html" %}
+{% load humanize %}
+{% load static %}
+{% block title %}
+    {{course.name}}: Discussion Forum
+{% endblock title %}
+{% block content %}
+    <div class="container">
+        <div>
+            <h2><center>{{course.name}}</center></h2>
+            <center>Discussion Forum</center>
+        </div>
+        <div class="pull-right">
```

Comment:
Also, it would be better if a back button is given to navigate back to the course modules for both moderator and student.